src/sentry/api/endpoints/relay/project_configs.py

```diff
@@ -43,19 +43,21 @@ def _post(self, request: Request):
         relay = request.relay
         assert relay is not None  # should be provided during Authentication
 
-        full_config_requested = request.relay_request_data.get("fullConfig")
+        if request.relay_request_data.get("globalConfig"):
+            metrics.incr("relay.project_configs.global.fetched")
```

Comment:
can i just use a new metric like this or do i have to declare it somewhere else?